### DISCUSSION TITLE
Scope / Filter using Grid Field values or Replicator Field values

### DISCUSSION THREAD

**Discussion by beatwiz** [statamic/cms#8523] [Q&A] [answered]
Hi! Is this achievable with core functionality? I was tasked to make a CP entries list filter that relies on fields within a Grid field. Any breadcrumbs are appreciated! Thanks :)

**Comment by beatwiz** [comment]
Im guessing the same logic as #8189 :) Although in my case index kinda makes it really hard.

**Comment by ryanmitchell** [comment, marked as answer]
Not yet. This is what https://github.com/statamic/cms/pull/8476 should achieve

**Reply by ryanmitchell** [reply]
And https://github.com/statamic/cms/pull/7454 now that I read your description again

**Reply by beatwiz** [reply]
Hi @ryanmitchell. I Ended up hacking a solution using keys for the time being. The dataset is relatively small, and its "just" 20 keys. Works super fast so no one minds. I'm leaving it like that till your PR is merged. Fantastic work there and both your PR's pointed me in the right direction. Thanks!! :)
